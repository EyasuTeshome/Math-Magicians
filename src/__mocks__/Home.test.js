import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';
import Home from '../components/Home/Home';

test('should render the home page', () => {
  const tree = render(<Router><Home /></Router>);
  expect(tree).toMatchSnapshot();
});
