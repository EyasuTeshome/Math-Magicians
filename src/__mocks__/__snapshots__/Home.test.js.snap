// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the home page 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="home_page_text_container"
      >
        <h2>
          Welcome to our page!
        </h2>
        <article
          class="text_article"
        >
          <p>
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce rhoncus justo magna, at sagittis ex bibendum non. Duis eget suscipit elit, eu rhoncus elit. Curabitur nisi urna, luctus sed est et, ornare mollis sem. Maecenas mattis euismod urna, id vehicula eros lacinia eget. Aliquam interdum ex id accumsan tristique. Sed eu blandit enim. In gravida, magna et semper egestas, arcu lacus ullamcorper sapien, sed finibus mauris ipsum quis urna. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Vestibulum id vulputate dui. Interdum et malesuada fames ac ante ipsum primis in faucibus. Sed ultricies ligula a facilisis.
          </p>
        </article>
        <article
          class="text_article"
        >
          <p>
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce rhoncus justo magna, at sagittis ex bibendum non. Duis eget suscipit elit, eu rhoncus elit. Curabitur nisi urna, luctus sed est et, ornare mollis sem. Maecenas mattis euismod urna, id vehicula eros lacinia eget. Aliquam interdum ex id accumsan tristique. Sed eu blandit enim. In gravida, magna et semper egestas, arcu lacus ullamcorper sapien, sed finibus mauris ipsum quis urna. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Vestibulum id vulputate dui. Interdum et malesuada fames ac ante ipsum primis in faucibus. Sed ultricies ligula a facilisis.
          </p>
        </article>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="home_page_text_container"
    >
      <h2>
        Welcome to our page!
      </h2>
      <article
        class="text_article"
      >
        <p>
          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce rhoncus justo magna, at sagittis ex bibendum non. Duis eget suscipit elit, eu rhoncus elit. Curabitur nisi urna, luctus sed est et, ornare mollis sem. Maecenas mattis euismod urna, id vehicula eros lacinia eget. Aliquam interdum ex id accumsan tristique. Sed eu blandit enim. In gravida, magna et semper egestas, arcu lacus ullamcorper sapien, sed finibus mauris ipsum quis urna. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Vestibulum id vulputate dui. Interdum et malesuada fames ac ante ipsum primis in faucibus. Sed ultricies ligula a facilisis.
        </p>
      </article>
      <article
        class="text_article"
      >
        <p>
          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce rhoncus justo magna, at sagittis ex bibendum non. Duis eget suscipit elit, eu rhoncus elit. Curabitur nisi urna, luctus sed est et, ornare mollis sem. Maecenas mattis euismod urna, id vehicula eros lacinia eget. Aliquam interdum ex id accumsan tristique. Sed eu blandit enim. In gravida, magna et semper egestas, arcu lacus ullamcorper sapien, sed finibus mauris ipsum quis urna. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Vestibulum id vulputate dui. Interdum et malesuada fames ac ante ipsum primis in faucibus. Sed ultricies ligula a facilisis.
        </p>
      </article>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
